app/controllers/admins/calculators_controller.rb

```diff
@@ -35,7 +35,7 @@ def create
     end
 
     def update
-      SelectPreferableService.new(calculator_params).select_preferable
+      Calculators::PreferableService.new(calculator_params).perform
```

Comment:
try to use `::Calculators::PreferableService` instead of `Calculators::PreferableService`